docs/customization.md

```diff
@@ -165,3 +165,19 @@ Content added to this file appears at the top of every page's main content betwe
 `_includes/nav_footer_custom.html`
 
 Any content added to this file will appear at the bottom left of the page below the site's navigation. By default an attribution to Just the Docs is displayed which reads, `This site uses Just the Docs, a documentation theme for Jekyll.`.
+
+### Custom Search Placeholder
+
+The placeholder text in the search bar (and its `aria-label`) is rendered by `_includes/search_placeholder_custom.html`, **after stripping HTML and whitespace**. By default, the content of the include is:
```

Comment:
I think it would be neater to use the same pattern as the other includes: put the file name as a paragraph right after the section heading, and use the active voice.

Reply:
I don't see the need to use boldface for the comment about stripping.

In any case, it isn't all white space that is stripped: only leading and trailing.

Reply:
You appear to have overlooked that last comment.